django_slack/tasks.py

```diff
@@ -1,9 +1,18 @@
-from celery import task
-
 from django.utils.module_loading import import_string
 
 from .app_settings import app_settings
 
-@task
-def send(*args, **kwargs):
```

Comment:
Yes, you are right, due to celery running in another process, tasks need to be import before executing. 
So, I guess I have to separate these two modules.

I manage to fixed it. 
So, do you want me open another PR? or just leave you to do this。